waspc/packages/studio/client/src/socket.ts

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { io } from "socket.io-client";
+import { Data } from "./types";
+
+export const socket = io("http://localhost:4000");
```

Comment:
It is hardcoded in the client and in the server 🤷 maybe we'll need to adjust this in the future, but I think it's okay for now since we are putting an [experimental] label on it